test/integration/scheduler_perf/scheduler_perf_legacy_test.go

```diff
@@ -440,7 +440,7 @@ func benchmarkScheduling(numExistingPods, minPods int,
 	b *testing.B) {
 	if b.N < minPods {
 		//lint:ignore SA3001 Set a minimum for b.N to get more meaningful results
-		b.N = minPods
+		b.N = minPods //nolint:staticcheck
```

Comment:
this is clamping the minimum of the b.N.

can we remove the old //lint:....` line?
why does the new static check need the pragma?

Reply:
There is an existing issue about staticcheck not respecting `//lint:ignore` when run from `golangci-lint`: https://github.com/golangci/golangci-lint/issues/741 but I don't know more beyond that. Using `//nolint:...` is what's suggested in the golangci-lint [docs](https://golangci-lint.run/usage/false-positives/).

Open to any suggestions on how to proceed. I originally left the old `//lint:ignore` pragmas in because they had more information than my new `//nolint:staticcheck` pragmas. The two things I am thinking of now are either either:
* deleting the `//lint:ignore`s but copying over their extra comments to the new `//nolint:staticcheck`s
* leaving the `//lint:ignore`s there so that if/when the issue I linked is fixed, we can go back to using those and we can delete the extra `//nolint:staticcheck`s in the future

Reply:
my vote is for the first option, with a comment of why this is being done (error message from the linter).

Reply:
Ok sounds good. I will update them all later tonight or tomorrow.